runtime/doc/msgpack_rpc.txt

```diff
@@ -413,4 +413,162 @@ of update.
 	The popupmenu is hidden.
 
 ==============================================================================
+7. Live Updates				      *live-updates* *rpc-live-updates*
+
+A dedicated API has been created to allow co-processes to be notified in
+real-time when the user changes a buffer in any way. (It is difficult and
+innefficient to do this using |TextChanged| and other autocommands.)
+
+							*live-updates-enabling*
+Setting Up~
+
+Use |jobstart()| to start up your co-process. >
```

Comment:
The norm will be that these will be used through the rplugin infrastructure, that allows multiple remote plugins to share a single python process (and similarly for other languages, like ruby), and therefore `jobstart()` should not be mentioned here (it shouldn't be anyway, as this part is already explain earlier in the same file) Even for a plugin than wants its own process I think the norm should be to use the host infractucture, which will manage event dispatch for the plugin instead of relying on `next_message()` (or an hypotetical `poll_message()`), this will obviously make it easier also to change the plugin to share the host later, if the need for a separate process later is removed.